### PR TITLE
Blog update: Simplify using SDS and new IKS features

### PR DESCRIPTION
**Pull request by rvennam**
Updates to blog https://istio.io/latest/blog/2020/alb-ingress-gateway-iks/
- Use SDS Gateways (Istio 1.6)..remove steps for creating file mounted certs and restarting gateway pod
- Use IKS nlb-dns --secret-namespace flag to create tls cert secret directly in istio-system namespace
- Appending issuer cert to the gateway tls crt is not needed for the ALB
- Provide ca certs to istio gateway using the <tls-cert>-cacert format

[ ] Configuration Infrastructure
[X ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by rvennam** [comment]
cc @vadimeisenbergibm

File: content/en/blog/2020/alb-ingress-gateway-iks/index.md [hunk 1]
```diff
@@ -239,9 +222,7 @@ You use the certificates and the keys provided to you for the ingress gateway an
           protocol: HTTPS
         tls:
           mode: MUTUAL
-          serverCertificate: /etc/istio/ingressgateway-certs/tls.crt
-          privateKey: /etc/istio/ingressgateway-certs/tls.key
-          caCertificates: /etc/istio/ingressgateway-ca-certs/trustid-x3-root.pem
+          credentialName: $INGRESS_GATEWAY_SECRET
```
Comment (vadimeisenbergibm) [inline content/en/blog/2020/alb-ingress-gateway-iks/index.md:225]: so ca certificates are not required here? Does it work without them?

Comment (rvennam) [reply content/en/blog/2020/alb-ingress-gateway-iks/index.md:225]: You don't need to specify the location of the ca certs in the Gateway yaml  if they are located in `$INGRESS_GATEWAY_SECRET-cacert`. See the last line in this section: https://istio.io/latest/docs/tasks/traffic-management/ingress/secure-ingress/#configure-a-mutual-tls-ingress-gateway

Comment (vadimeisenbergibm) [reply content/en/blog/2020/alb-ingress-gateway-iks/index.md:225]: I see, did not know that.

**Comment by vadimeisenbergibm** [comment]
Fine with me.

**Pull request merged**